.graalvm/resource-config.json

```diff
@@ -1,43 +1,43 @@
 {
   "resources": [
     {
-      "pattern": "\\Qbamm/characteristic/2.0.0/characteristic-definitions.ttl\\E"
```

Comment:
Tip: all the samm patterns can be folded into one expression like this: https://github.com/eclipse-esmf/esmf-sdk/blob/main/tools/samm-cli/src/main/resources/META-INF/native-image/org.eclipse.esmf/samm-cli/resource-config.json#L11